x-pack/plugins/osquery/cypress/e2e/all/add_integration.cy.ts

```diff
@@ -42,7 +42,7 @@ describe('ALL - Add Integration', () => {
   });
 
   beforeEach(() => {
-    login();
+    cy.login('elastic');
```

Comment:
why do we have to login as `elastic` on ESS?

Reply:
changed to soc_manager in general + added a 'viewer' role. But to answer your question: we didn't have to change to elastic in ESS, it was done just for unification.

Reply:
ok this specific login is still elastic, as it was login() before. because we need more access to fleet here. the other tests are changed back into soc_manager.